cast/src/lib.rs

```diff
@@ -405,6 +405,10 @@ where
         Ok(datetime.format("%a %b %e %H:%M:%S %Y").to_string())
     }
 
+    pub async fn timestamp<T: Into<BlockId>>(&self, block: T) -> Result<U256> {
+        Ok(Cast::block_field_as_num(self, block, String::from("timestamp")).await?)
```

Comment:
is this just a convention of the codebase, but otherwise these two are equivalent? I'll make this change, but just noting that I used `String::from` since that's what was used in the `age` and `base_fee` methods above (though, `age` also uses `to_string()`, so I guess things are a bit inconsistent)

Reply:
is this just a convention of the codebase, but otherwise these two are equivalent? I'll make this change, but just noting that I used `String::from` since that's what was used in the `age` and `base_fee` methods above (though, `age` also uses `to_string()`, so I guess things are a bit inconsistent)